modules/dynamic-provisioning-change-default-class.adoc

```diff
@@ -21,7 +23,7 @@ $ oc get storageclass
 [source,terminal]
 ----
 NAME                 TYPE
-gp2 (default)        kubernetes.io/aws-ebs <1>
+gp3 (default)        kubernetes.io/aws-ebs <1>
```

Comment:
In 35 lines,the gp2 should be updated to gp3 too
`oc patch storageclass gp2 -p '{"metadata": {"annotations": {"storageclass.kubernetes.io/is-default-class": "false"}}}'`

Reply:
Here we just show how to update the default storageclass, we could even use common example like "Golden/Silvery" or "fast/slow", then no need to update in the further to map the real storageclass in OCP.
But we have to make sure the storageclass name specified in cmd below should be the same as example as mentioned by @yingzhanredhat

Reply:
Done.